docs/documentation/baremetal-upi/index.md

```diff
@@ -126,4 +126,60 @@ Click on Create Hyperconverged button, all the defaults should be fine.
 
 ## Providing storage
 
```

Comment:
Would you mind adding a üaragraph about what features require shared storage vs what implications local storage has?

Doesn#t need to be long and does not need to be perfect, but at least some words.